pkg/operation/botanist/component/kubeapiserver/monitoring.go

```diff
@@ -184,6 +184,17 @@ const (
 
   - record: shoot:kube_apiserver:sum_by_pod
     expr: sum(up{job="` + monitoringPrometheusJobName + `"}) by (pod)
+  ### API failure rate ###
+  - alert: ApiserverRequestsFailureRate10mins
+    expr: max(sum by(instance,resource,verb) (rate(` + monitoringMetricApiserverRequestTotal + `{code=~"5.."}[10m])) / sum by(instance,resource,verb) (rate(` + monitoringMetricApiserverRequestTotal + `[10m]))) * 100
+    for: 30m
+    labels:
+      severity: warning
+      type: seed
+      visibility: owner
+    annotations:
+      description: The Apiserver requests failure rate exceed 0.3%.
+      summary: Kubernetes API server failure rate is high
```

Comment:
There are several issues here, please review / double check.

- please make sure the fields and texts are consistent (30m vs 10mins)
- is 0.3% high? maybe you meant 30%? Anyways, the comparison with this value is missing in the promql expression. Without such a comparison (`>0.3`) the alert would always fire
- please watch out for typo-s (exceed, exceeds)

please consider using the local setup to test your changes: https://github.com/gardener/gardener/blob/master/docs/deployment/getting_started_locally.md